setup.py

```diff
@@ -27,10 +27,10 @@ def run_tests(self):
     "packages": ["pygerduty"],
     "scripts": ["bin/grab_oncall.py"],
     "description": "Python Client Library for PagerDuty's REST API",
-    "author": "Gary M. Josack",
-    "maintainer": "Gary M. Josack",
-    "author_email": "gary@dropbox.com",
-    "maintainer_email": "gary@dropbox.com",
+    "author": "Mike Cugini",
```

Comment:
:'(

Reply:
Oh whoops, my vim replace was a little too aggressive :(. Will fix today for next version.